app/src/componentLibrary/autocompletes/withAsyncLoading.jsx

```diff
@@ -47,16 +49,21 @@ export const WithAsyncLoading = (AutocompleteComponent) =>
     }
 
     debouncedFetch = debounce((inputValue) => {
-      const { getURI, makeOptions, filterOption } = this.props;
+      const { getURI, getRequestParams, makeOptions, filterOption } = this.props;
+
       if (this.cancelToken) {
         this.cancelToken();
       }
+
       const value = (inputValue || '').trim();
       const uri = getURI(value);
+      const requestParams = getRequestParams(value);
```

Comment:
As I can see, If getRequestParams is not provided, the result of this function invocation will be `undefined`.

Reply:
Fixed in the scope of https://github.com/reportportal/service-ui/pull/3670